pyaedt/examples/downloads.py

```diff
@@ -15,7 +15,7 @@ def delete_downloads():
 
 def _get_file_url(directory, filename):
     if not filename:
-        return EXAMPLE_REPO + '/'.join([directory])
+        return EXAMPcoordinateLE_REPO + '/'.join([directory])
```

Comment:
We should consider using `os.path.join` in the future.  This works, but it tends to be brittle.